src/pages/MyProfile/MyProfile.test.js

```diff
@@ -39,19 +43,19 @@ describe('Rendering', () => {
     expect(input).toBeInTheDocument()
   })
 
-  it('should render the email input', async () => {
-    const { getByLabelText } = renderMyProfile()
-    const input = getByLabelText('Email')
+  // it('should render the email input', async () => {
```

Comment:
use it.skip rather than commenting out tests.

It gives us more visibility in the test report.

Reply:
The commented out test code relates to the commented out component code.  Since we're deleting the component commented code, I'll go ahead and just delete this code respectively, instead of skipping it, as this would be the most accurate overview of the tests.